.jvmopts

```diff
@@ -1,8 +1,5 @@
--server 
 -Xms512M 
 -Xmx4000M 
```

Comment:
I suggest to increase heap because it isn't enough to build SN.

See example of such attempts at https://github.com/scalatest/scalatest/pull/1904 or https://github.com/scalatest/scalatest/pull/1929

Reply:
this is 1929, you meant 1904 and...?

Reply:
oh, I see it now, #1928

Reply:
@SethTisue and this is that only I know!